packages/cli/test/unit/lib/redis/shared.unit.test.ts

```diff
@@ -0,0 +1,54 @@
+import {stdout} from 'stdout-stderr'
```

Comment:
i'm not sure why this is registering as a new file.. it already exists?
https://github.com/heroku/cli/blob/prerelease/9.0.0-alpha/packages/cli/test/unit/lib/redis/shared.ts

oh i think i answered my own question. the original one (which i authored) is missing the `.unit.test.ts` extension. maybe delete that one or rename it or whatever?

Reply:
Oh, I didn't noticed it. I will take a look and see if there are any substantial differences, and remove one of them.